### PR TITLE
doc: add run-as-daemon.md

### PR DESCRIPTION
**Pull request by yqlbu**
## Summary

As the title suggests.

### CODE REVIEW

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -38,13 +38,14 @@ make GOFLAGS="-buildvcs=false" CC=clang
...
-Download [geoip.dat](https://github.com/v2ray/geoip/releases/latest) and [geosite.dat](https://github.com/v2fly/domain-list-community/releases/latest) to `/usr/local/share/dae/`.
+For traffic splitting, Dae relies on the following data sources, [geoip.dat](https://github.com/v2ray/geoip/releases/latest) and [geosite.dat](https://github.com/v2fly/domain-list-community/releases/latest).
 
-```
+```shell
 mkdir -p /usr/local/share/dae/
 pushd /usr/local/share/dae/
 curl -L -o geoip.dat https://github.com/v2ray/geoip/releases/latest/download/geoip.dat
 curl -L -o geosite.dat https://github.com/v2ray/domain-list-community/releases/latest/download/dlc.dat
+curl -L -o config.dae https://github.com/v2rayA/dae/raw/main/example.dae
```
Comment (mzz2017) [inline docs/getting-started/README.md:48]: This file should be downloaded to `/etc/dae/`. Or you can move it after `popd`.

File: docs/getting-started/run-as-daemon.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+```bash
+# download the sample systemd.service
+sudo curl -L -o /etc/systemd/system/dae.service https://github.com/v2rayA/dae/raw/main/install/dae.service
+
+# reload and restart daemon to take effect
+sudo systemctl daemon-reload
+sudo systemctl enable dae --now
+sudo systemctl status dae
+```
```
Comment (mzz2017) [inline docs/getting-started/run-as-daemon.md:20]: Add some notes about config file (for examples: move the config file downloaded just now to `/etc/dae/`)

**Review by mzz2017 — CHANGES_REQUESTED** [review]

File: docs/getting-started/README.md (hunk 1 above)
Comment (yqlbu) [reply docs/getting-started/README.md]: Resolved in [eb8dc6c](https://github.com/v2rayA/dae/pull/25/commits/eb8dc6c4554d89391f4f7d20b9b4a0000a98126e)

File: docs/getting-started/run-as-daemon.md (hunk 2 above)
Comment (yqlbu) [reply docs/getting-started/run-as-daemon.md:32]: Resolved in [eb8dc6c](https://github.com/v2rayA/dae/pull/25/commits/eb8dc6c4554d89391f4f7d20b9b4a0000a98126e)

**Comment by mzz2017** [comment]
LGTM

**Comment by mzz2017** [comment]
Thanks for your contribution!

**Pull request merged**